kubos/utils/constants.py

```diff
@@ -34,3 +34,4 @@
 GLOBAL_TARGET_PATH  = os.path.join('/', 'usr', 'local', 'lib', 'yotta_targets')
 GLOBAL_MODULE_PATH  = os.path.join('/', 'usr', 'local', 'lib', 'yotta_modules')
 
+SHOW_NUMBER_MINOR_VERSIONS = 0
```

Comment:
This should probably get renamed too, since 'minor' here is still referring to the CD-generated version level, correct?